.gitignore

```diff
@@ -5,5 +5,7 @@
 node_modules
 npm-debug.log
 coverage
-
+cypress/videos
```

Comment:
what if we have single folder to keep test data instead of multiple folders? something like `cypress/test_data`

Reply:
we have a single folder for test data, cypress/integration. The videos, screenshot and log folders contains the output from failed tests, hence added to gitignore